src/lib/docs/AccountKit/BuyRamMethod.md

```diff
@@ -0,0 +1,47 @@
+---
+title: BuyRam (Method)
+description: Used to generate an action to acquire RAM using a specific amount of tokens.
+category: ContractKit
+published: true
+order: 5
+---
+
+# BuyRam
+
+The `buyRam` method allows accounts to purchase RAM using a specific amount of tokens. It returns an [Action](/docs/antelope/action) instance detailing the RAM purchase request.
```

Comment:
Should all of these explanations of actions also indicate they still need to be executed using a Session?

It says it will return an action, but maybe it also needs to include "the action then needs to be performed using something like a Session" (and include links on performing transactions, beyond just creating them). 

Not sure, but whatever we decide should also be applied to all these "action-creation methods".

Reply:
That's mentioned at the top of the page, but adding it here probably make sense.

Reply:
Yeah, I usually mention that in the `Return Value` section. Are you saying that it should be mentioned in both places are just in the function's explanation?